Mathlib/LinearAlgebra/Matrix/PosDef.lean

```diff
@@ -165,4 +166,22 @@ def InnerProductSpace.ofMatrix {M : Matrix n n 𝕜} (hM : M.PosDef) :
   InnerProductSpace.ofCore _
 #align matrix.inner_product_space.of_matrix Matrix.InnerProductSpace.ofMatrix
 
+section ConjTransposeMul
+
+variable {m : Type _} [Fintype m]
```

Comment:
Isn't there already an `m` at the top of the file?

Reply:
I think it is in a separate namespace. There are three namespaces in this file